modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/service/impl/KBArticleServiceImpl.java

```diff
@@ -891,9 +892,9 @@ else if (portletId.equals(KBPortletKeys.KNOWLEDGE_BASE_DISPLAY)) {
 	}
 
 	private String _exportToRSS(
-		String rssDisplayStyle, String rssFormat, String name,
-		String description, String feedURL, List<KBArticle> kbArticles,
-		ThemeDisplay themeDisplay) {
+		String description, String displayStyle, String feedURL,
+		List<KBArticle> kbArticles, String name, ThemeDisplay themeDisplay,
+		String type, double version) {
```

Comment:
Alphabetic order is fine for private *constructors*; for private methods I'm not so sure. I'd make the order consistent with the other RSS methods.

Reply:
I'm leaving the parameters order as it was before my changes. I only replaced the old `rssFormat` parameter with the new parameters `type` and `version`.